### PR TITLE
refactor(*): Upgrade to Swift 5

### PR DESCRIPTION
**Pull request by emrcftci**

### CODE REVIEW

**Comment by Zandor300** [comment]
@amrit-1901 I think this can be merged.

File: iOS/WAStickersThirdParty/StickerPackViewController.swift [hunk 1]
```diff
@@ -93,6 +93,8 @@ class StickerPackViewController: UIViewController, UICollectionViewDataSource, U
         let buttonLandscapeSize: CGSize = CGSize(width: 250.0, height: 50.0)
         let buttonBottomMargin: CGFloat = 20.0
 
+        guard let view = view else { return }
```
Comment (amrit-1901) [inline iOS/WAStickersThirdParty/StickerPackViewController.swift]: Why is this needed?

Comment (emrcftci) [reply iOS/WAStickersThirdParty/StickerPackViewController.swift]: Because `view`'s type is implicitly-unwrapped optional UIView. 
If we don't unwrap the view property warnings appear in the following lines.


Screen Shot 2020-01-28 at 10 51 19
